youtube/reviews.md

```diff
@@ -29,11 +29,11 @@ https://www.youtube.com/eddiejaoude?sub_confirmation=1
 
 Follow on other socials for behind the scenes footage, join discord to continue the conversation...
 
-http://eddiejaoude.io 
-https://github.com/eddiejaoude
-http://youtube.com/eddiejaoude
-http://twitter.com/eddiejaoude
-http://instagram.com/eddiejaoude
-http://linkedin.com/in/eddiejaoude
-https://discord.com/invite/jZQs6Wu
-http://github.com/EddieHubCommunity
+https://www.eddiejaoude.io
+https://www.github.com/eddiejaoude
```

Comment:
I'm not sure if we want to use `www` for all the links?
(In current context, IMO it's easier to just use the domain directly, esp. since the name sticks and is easier to use that way)

Reply:
IDK for me both ways are ok :)

Reply:
I opened a small chat discussion on discord, to get some feedback on this from the community - https://discord.com/channels/699608417039286293/738541096098988072/846021789074718751

Reply:
@Vyvy-vi if we don't use www for https://eddiejaoude.io It's giving error. 
![Screenshot 2021-05-24 171723](https://user-images.githubusercontent.com/60481830/119343356-f3c6ec00-bcb3-11eb-8744-1028d97c143b.png)

Reply:
yes, the `eddiejaoude.io` and `eddiehub.org` sites need the `www`, if we want to link the `https` link.
However, maybe we can omit using www everywhere?

Reply:
yeah! your right but I'm not sure about which one is effective(either with _www_ or _www_)

Reply:
adding www is better. But it doesn't work for discord.eddiehub.org

Reply:
I'm not sure objectively if there is any difference if we're simply linking. It would be of more discussable "utility", if we were hosting our own site, however for sites like github and youtube, currently `www` is often omitted...
I'm not really sure what would be preferred, but imo it's better to be succinct with URLs _when linking them to other people, or posting them_.

Here's what Eddie's replied with -
> I prefer without the `www`, the kartra links should work without and then redirect to the www

But, since we are choosing a sorta-convention here, maybe we should look into how Eddie's links are usually formatted and set these links up to be consistent with that?

Reply:
Ok I see the point using no `www` I think I will check all links working properly without `www`.
Think this is also the better solution due this is written in plain text, the urls should not unnecessary long 🚀 
This I think would be the preferred way to go :)

Reply:
All should work properly, except eddiehub.org and eddiejaoude.io (those need `www`)

Reply:
I've done the changes